clang/test/Driver/aix-ld.c

```diff
@@ -1120,3 +1120,18 @@
 // RUN:        -c \
 // RUN:   | FileCheck --check-prefixes=CHECK-K-UNUSED %s
 // CHECK-K-UNUSED: clang: warning: -K: 'linker' input unused [-Wunused-command-line-argument]
+
+// Check No Sanitizer on 32-bit AIX
+// This test verifies that the linker doesn't include '-latomic' when no sanitizers are enabled 
+// RUN: %clang -target powerpc-ibm-aix -m32 %s -### 2>&1 \
```

Comment:
`-target ` has been deprecated for a long time though without a warning. Use `--target=` for new tests